cfme/services/myservice/ssui.py

```diff
@@ -41,6 +41,7 @@ def is_displayed(self):
     policy_btn = SSUIDropdown('Policy')
     lifecycle_btn = SSUIDropdown('Lifecycle')
     power_operations = SSUIDropdown('Download')
```

Comment:
Can you change this to power_operations = SSUIDropdown('Power Operations')

Reply:
will do :)

Reply:
done